pkg/cloudprovider/kubevirt/loadbalancer.go

```diff
@@ -184,11 +184,13 @@ func (lb *loadbalancer) createLoadBalancerService(lbName string, service *corev1
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      lbName,
 			Namespace: lb.namespace,
+			Annotations: service.Annotations,
```

Comment:
There's a security problem: What if the seed cluster has some MetalLB address pools that are not intended to be used by shoot clusters?

Maybe there should be a set of allowed annotations with allowed values.

Reply:
As discussed on slack, I agree here, but believe the administrative controls on this should be in the client (such as Gardener) and not here.

Reply:
I agree with @briantopping here. Since the CCM requires a ServiceAccount that is allowed to create and modify Services, a user could always use this serviceAccount to insert annotations to the service manually if he wishes.

Even if CCM would forbid overwriting the metallb IPs via annotations, it would still be possible to manually modify the Service and insert an annotation to use metallb addresses that where not intended to be used.